### PR TITLE
retrieve-image-sboms: Document License information and source code URLs

### PR DESCRIPTION
**Pull request by xnox**
Add documentation about license information and source code URLs in Chainguard Images SBOM.

### CODE REVIEW

**Review by SharpRake — APPROVED** [review]
I'm giving this my approval so it can be published any time but would appreciate @xnox (and @imjasonh + @jonjohnsonjr as requested) giving it another look to make sure I didn't misinterpret anything or cut anything important.

Thanks again for putting this together!

File: content/chainguard/chainguard-images/working-with-images/retrieve-image-sboms/index.md [hunk 1]
```diff
@@ -74,51 +74,53 @@ You can use the drop-down menu above the table to select which version and archi
...
 
-Source code references are encoded in the [external references](https://spdx.github.io/spdx-spec/v2.3/package-information/#721-external-reference-field) field, using [external repository identifiers](https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f35-purl) in PURL format. PURL [specification](https://github.com/package-url/purl-spec/blob/master/PURL-SPECIFICATION.rst) allows for many schemes and types. The following PURLs are used in Chainguard SPDX SBOM:
+These source code references are encoded in the [external references](https://spdx.github.io/spdx-spec/v2.3/package-information/#721-external-reference-field) field, using [external repository identifiers](https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f35-purl) in the package URL (purl) format. The [purl specification](https://github.com/package-url/purl-spec/blob/master/PURL-SPECIFICATION.rst) allows for various different schemes and types. 
 
- * `pkg:apk` denotes binary package origin, name, full version number with epoch and architecture. Example:
+The following The following purls are used in Chainguard SPDX SBOM:
```
Comment (xnox) [inline content/chainguard/chainguard-images/working-with-images/retrieve-image-sboms/index.md:83]: duplicate `the folloowing the following`

**Comment by xnox** [comment]
> I'm giving this my approval so it can be published any time but would appreciate @xnox (and @imjasonh + @jonjohnsonjr as requested) giving it another look to make sure I didn't misinterpret anything or cut anything important.
> 
> Thanks again for putting this together!

It's great, just the one duplicate typo duplicate typo, see inline comment above.

**Comment by SharpRake** [comment]
> > I'm giving this my approval so it can be published any time but would appreciate @xnox (and @imjasonh + @jonjohnsonjr as requested) giving it another look to make sure I didn't misinterpret anything or cut anything important.
> > Thanks again for putting this together!
> 
> It's great, just the one duplicate typo duplicate typo, see inline comment above.

ACK - nice catch!

**Pull request merged**